e2e/app/component/sidebar-content.ts

```diff
@@ -0,0 +1,64 @@
+
+import {Page} from 'puppeteer';
+import HelpSidebar from 'app/component/help-sidebar';
+import ClrIconLink from 'app/element/clr-icon-link';
+import {getPropValue} from 'utils/element-utils';
+import ReactSelect from 'app/element/react-select';
+import {waitWhileLoading} from 'utils/waits-utils';
+
+
+const defaultXpath = '//*[@data-test-id="sidebar-content"]';
+
+export enum ReviewStatus {
+  Excluded = 'Excluded',
+  Included = 'Included',
+  NeedsFurtherReview = 'Needs Further Review '
+}
+
+export default class SidebarContent extends  HelpSidebar {
+  
+
+    constructor(page: Page, xpath: string = defaultXpath) {
+      super(page, xpath);
+    }
+
+    async getParticipantID(): Promise<string> {
+        const selector = `${this.getXpath()}//div[1][text()="Participant "]`;
+        const pID = await this.extractParticipantDetails(selector);
+        console.log(`Participant ID: ${pID}`);
+        return pID;
+    }
+
+    async getParticipantDOB(): Promise<string> {
+        const selector = `${this.getXpath()}//*[contains(normalize-space(text()), "DOB")]/parent::*//span/following-sibling::text()`;
+        const pidDob = await this.extractParticipantDetails(selector);
+        console.log(`DOB: ${pidDob}`);
+        return pidDob;
+    }
+
+    
+    private async extractParticipantDetails(selector: string): Promise<string> {
+      const elemt = await this.page.waitForXPath(selector, {visible: true});
+      const textContent = await getPropValue<string>(elemt, 'textContent');
+      // const regex = new RegExp(/\d{1,3}(,?\d{3})*/); // Match numbers with comma
+       const regex = new RegExp(/\d+/);
+      return (regex.exec(textContent))[0];
+    }
+
+    /**
+     * select participant status.
+     * @param {ReviewStatus}  status
+     */
+    async selectReviewStatus(status: ReviewStatus): Promise<string> {
+      const selectMenu = new ReactSelect(this.page, {name:'Choose a Review Status for Participant'});
+      await selectMenu.selectOption(status);
+      await waitWhileLoading(this.page);
+      return selectMenu.getSelectedOption();
+    }
+  
+    async getAnnotationsButton(): Promise<ClrIconLink> {
+      return ClrIconLink.findByName(this.page, {iconShape: 'plus-circle'});
```

Comment:
this is not correct selector. if there is a `plus-circle` icon outside of sidebar, it will likely find the wrong icon.

Reply:
![image](https://user-images.githubusercontent.com/48066285/99705708-b6ad4300-2a5f-11eb-832f-4086a912f07d.png)
this plus-circle icon is within the `<div data-test-id="sidebar-content" >`. Do you think it'll still not work?

Reply:
problem is that you're not using sidebar anywhere as parent in selector

Reply:
ok! got it. Will update it. Thanks!

Reply:
done!